modules/cli/bin/xviz

```diff
@@ -35,14 +35,9 @@ if (
 
   require('source-map-support').install();
 
-  // Registers aliases for virtual packages in this module
-  var moduleAlias = require('module-alias');
-  moduleAlias.addAliases({
-    '@xviz/builder': resolve(__dirname, '../../builder/src'),
-    '@xviz/parser': resolve(__dirname, '../../parser/src'),
-    '@xviz/schema': resolve(__dirname, '../../schema/src'),
-    '@xviz/cli': resolve(__dirname, '../../cli/src')
-  });
+  const ALIASES = require('../../../aliases');
```

Comment:
This is not going to work when it's published.

Reply:
Yeah the XVIZ_DEV environment variable checks farther up in the file limit this to only be used when working on the tool not for users, let me comment things better.